docs/pages/management/guides/joining-nodes-azure.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: Joining Nodes via Azure Managed Identity
+description: Use the Azure join method to add Nodes to your Teleport cluster on Azure
+---
+
+This guide will explain how to use the **Azure join method** to configure
+Teleport Nodes and Proxy Service instances to join your Teleport cluster without
+sharing any secrets when they are running in an Azure Virtual Machine.
+
+The Azure join method is available in Teleport 12.1+. It is available to any
+Teleport Node or Proxy running in an Azure Virtual Machine. The Virtual Machine
+must have a
+[Managed Identity](https://learn.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview)
+assigned to it with permission to read virtual machine info.
+
+<Details
+scope={["oss", "enterprise", "cloud"]}
+scopeOnly
+opened
+title="Another Azure Node joining method"
+>
+
+You can also configure Nodes running in Azure to join a cluster via **secret tokens**,
+which is useful when you don't want to rely on Azure-specific APIs.
+Read more in the following guide:
+[Adding Nodes to the cluster](../admin/adding-nodes.mdx)
+
+</Details>
+
+<Notice type="warning" scope={["oss", "enterprise"]}>
+
+The Azure join method will not work if TLS is terminated at a load balancer in
+front of your Teleport Proxy Service unless the Node using this method is
+connecting directly to the Auth Service.
+
+</Notice>
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- An Azure Virtual Machine to act as a Teleport Node, with the Teleport binary
+  installed.
+
+(!docs/pages/includes/tctl.mdx!)
+
+## Step 1/4. Set up Managed Identity.
+Every virtual machine hosting a Node or Proxy using the Azure method to join
+your Teleport cluster needs a Managed Identity assigned to it. The identity 
+requires at least the `Microsoft.Compute/virtualMachines/read` permission.
+
+To create a managed identity with an appropriate role, first
+[create a custom role](https://learn.microsoft.com/en-us/azure/role-based-access-control/custom-roles)
+with the `Microsoft.Compute/virtualMachines/read` permission.
+Then [create a new user-assigned managed identity](https://learn.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/how-manage-user-assigned-managed-identities?pivots=identity-mi-methods-azp#create-a-user-assigned-managed-identity)
+and [assign it your custom role](https://learn.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/how-manage-user-assigned-managed-identities?pivots=identity-mi-methods-azp#manage-access-to-user-assigned-managed-identities).
+Note the client ID of the identity for Step 3. Finally,
+[assign the identity to the VM that will join your Teleport cluster](https://learn.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/qs-configure-portal-windows-vm).
+
+<Admonition
+  type="warning"
+  title="IMPORTANT"
+>
+As part of authentication, Teleport will be able to do anything allowed by the
```

Comment:
> As part of authentication, Teleport will be able to do anything allowed by the identity. 

Does this mean that, when Teleport authenticates to Azure, it will be able to list all virtual machines (using the `read` permissions described above)?

Reply:
Yes.

Reply:
Do we have any instructions we can provide for limiting the managed identity's permissions to only what is necessary? We already instruct the reader to scope the managed identity to a resource group/region, for example.

Reply:
Limiting the scope and permissions should be enough. This warning is more geared towards readers using an identity other than the one created in the guide.

Reply:
Makes sense, thanks!